Code/GraphMol/SmilesParse/smatest.cpp

```diff
@@ -2770,6 +2770,15 @@ void testSmartsStereoBonds() {
       TEST_ASSERT(bnd->getStereo() == Bond::STEREONONE);
     }
   }
+  {
+    // A weird way of writing C/C=C/O:
```

Comment:
nice test.

Reply:
It's the exact example that made me notice something was wrong :)